setup.py

```diff
@@ -142,7 +142,7 @@
         'SLIST = obspy.io.ascii.core',
         'PICKLE = obspy.core.stream',
         'CSS = obspy.io.css.core',
-        'DATAMARK = obspy.io.datamark.core',
+        'WIN = obspy.io.win.core',
```

Comment:
I'd suggest to keep this as-is... I don't know many users of this routine anyway, and I've modified the docstring so that a simple googling should show DATAMARK in the io.win explanation